### ISSUE TITLE
Docs are unreadable

### ISSUE THREAD

**Issue by fmvilas** [asyncapi/vs-asyncapi-preview#117]
#### Describe the bug
Docs are unreadable. See screenshot.

#### How to Reproduce
Not sure how to reproduce it. I'm using the Material Theme, which is probably changing the default color.

![Captura de Pantalla 2022-08-30 a las 11 02 58](https://user-images.githubusercontent.com/242119/187396704-d1ac4fb9-e2dc-414f-839c-e2f66dada64d.png)

---

This problem should probably be addressed at the AsyncAPI React component. If we make sure this is covered there, it won't be a problem here or in any other editor.

**Comment by derberg** [comment]
I think it is related to any dark theme in VSCode, because the light theme looks good.

**Comment by magicmatatjahu** [comment]
Yeah, we don't "define" color of text in react-component, so dark themes will override default one. I will fix that.

**Comment by ivangsa** [comment]
hi @magicmatatjahu 
is this fixed in any version of the react-component?

**Comment by leigh-johnson** [comment]
:wave: I'm seeing this too, using the Dark Molokai theme. What's needed to move this forward? Is the only workaround to change to a light theme?

![Screenshot from 2022-11-27 10-21-27](https://user-images.githubusercontent.com/2601819/204152822-bd9dc93f-ad6e-4e11-b1a1-131031b2c866.png)

**Comment by vancejc-mt** [comment]
Would be great if this could be fixed to ignore the color theme, I too use Monokai dark which makes this unreadable.
